Specs/Scene/ExpressionSpec.js

```diff
@@ -126,6 +126,30 @@ defineSuite([
         }).toThrowRuntimeError();
     });
 
+    it('evaluates variable to undefined if feature is undefined', function() {
+        var expression = new Expression('${height}');
+        expect(expression.evaluate(undefined)).toBeUndefined();
+
+        expression = new Expression('${vector.x}');
+        expect(expression.evaluate(undefined)).toBeUndefined();
+
+        expression = new Expression('${feature}');
+        expect(expression.evaluate(undefined)).toBeUndefined();
+
+        expression = new Expression('${feature.vector}');
+        expect(expression.evaluate(undefined)).toBeUndefined();
+
+        expression = new Expression('${vector["x"]}');
+        expect(expression.evaluate(undefined)).toBeUndefined();
+
+        expression = new Expression('${feature["vector"]}');
+        expect(expression.evaluate(undefined)).toBeUndefined();
+
+        // Evaluating inside a string is an exception. "" is returned instead of "undefined"
```

Comment:
Why is this an exception? It looks like we determine the behavior in the [previous test](https://github.com/AnalyticalGraphicsInc/cesium/pull/7255/files#diff-8393c04714ea81aa7c7e54f75becfc78L118) for what do when printing an undefined property.

Reply:
Oh just an exception in reference to the name of the test "evaluates variable to undefined if feature is undefined"